left-right-tlcache/src/lib.rs

```diff
@@ -209,3 +209,368 @@ macro_rules! make_thread_local_readhandle_cache {
         }
```

Comment:
Typo in the commit message "ests -> tests"

Reply:
Fixed